packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -2176,11 +2156,14 @@ export async function _makeBuyOrder({
   if (!asset.asset_contract) {
     throw new Error('contract address not defined within asset')
   }
+
   return {
+    // @ts-ignore
```

Comment:
Might be able to remove these ignores since in the other PR I changed the type to BigNumber | string

https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/4115/files

Reply:
Good point, forgot about that PR